addons/vagrant/wireless/Vagrantfile

```diff
@@ -0,0 +1,53 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Require YAML module
+require 'yaml'
+ 
+# Read YAML file with box details
+inventory = YAML.load_file('inventory/hosts')
+networks = inventory['all']['vars']['networks']
+mgmt_ip = inventory['all']['children']['wireless']['hosts']['wireless01']['mgmt_ip']
+
+
+Vagrant.configure("2") do |config|
+
+  wbid = 1
+  offset = wbid * 100
+
+  ##### DEFINE VM for wireless01 #####
+  config.vm.define "wireless01" do |device|
+
+    device.vm.hostname = "wireless01"
+    device.vm.box = "generic/debian10"
```

Comment:
If possible, I prefer to use `debian/buster64` image in order to have same image on all Debian hosts.

Reply:
Yes, I have started with this one then I have changed but I do not remember why....
I am wondering if it was a problem of kernel and the availability of modprobe `mac80211_hwsim` on this version.

Reply:
Finally, it looks good with `debian/buster64`.